### ISSUE TITLE
fontify with no options stopped working

### ISSUE THREAD

**Issue by avkonst** [westracer/fontify#13]
Hi, thanks for the upgrade to null safety.
I previously used fontify as the following:
```
>  flutter pub run fontify
```
and it read all settings from fontify section in my pubspec.yaml file:
```
fontify:
  input_svg_dir: "assets/svgs/"
  output_font_file: "assets/fonts/AssetIcons.otf"

  output_class_file: "lib/icons.dart"
  class_name: "AssetIcons"
  indent: 4
  font_name: "AssetIcons"
  normalize: true
  ignore_shapes: true

  recursive: true
  verbose: false
```

And it worked.

Now I need to run it as the following in order to get the same result:
```
flutter pub run fontify assets/svgs/ assets/fonts/AssetIcons.otf
```
Basically, it has got now 2 mandatory arguments which are also in the config file.

Is it possible to put the previous behaviour back?

**Comment by westracer** [comment]
Hello. The previous behavior should be back now - the fix has been released in 0.3.0-nullsafety.1

**Comment by avkonst** [comment]
Thank you for your quick update!
